### PR TITLE
Add Course creation page

### PR DESCRIPTION
**Pull request by m-e-l-u-h-a-n**
- An Extra component ta-input has been used for ease.
@king-11 
I have used Object.assign() for copying the objects in ta input I may replace them with spread operators in some later prs if you want.

### CODE REVIEW

**Comment by king-11** [comment]
I will check it soon as far as spread is considered we can make optimization + switch to a cleaner and new methods in final stage.

File: pages/createcourse.vue [hunk 1]
```diff
@@ -0,0 +1,287 @@
+<template>
+  <div>
+    <v-row row wrap justify="center">
+      <v-col cols="12" md="10">
+        <v-card elevation="2">
+          <v-card-title class="info--text">
+            Create Assignment
+          </v-card-title>
```
Comment (MadhavaDasari) [inline pages/createcourse.vue:8]: I think it's "Create Course"  😅

**Review by MadhavaDasari — COMMENTED** [review]
Course is getting created ( i.e snackbar will successful msg ) even if the TA's details are not entered properly  ..do check it . ( when "save course" button is pressed without pressing "save Ta" ) 

-  Fields of Ta's are getting outlined ( sm size ) only  when  the edit button is click ( it that intended ? 🙄 )

-  Other things looks good to me 👍 

>

File: pages/createcourse.vue (hunk 1 above)
Comment (m-e-l-u-h-a-n) [reply pages/createcourse.vue:8]: Yes, :smile:

**Comment by m-e-l-u-h-a-n** [comment]
> Course is getting created ( i.e snackbar will successful msg ) even if the TA's details are not entered properly ..do check it . ( when "save course" button is pressed without pressing "save Ta" )
> 
>     * Fields of Ta's are getting outlined ( sm size ) only  when  the edit button is click ( it that intended ? roll_eyes )
> 
>     * Other things looks good to me +1
> 
> 
> >

- Yes, outlined fields are intentionally shown in small screens only cuz big big boxes in large screens appear vary awkward.
- when a new ta is added, it gets added with primary details(unititled name and dummy email) anytime one tries to edit the ta, until he/she saves the ongoing edit ta's previous details will only be checked upon saving the course(like if one has not commited changes for the ongoing edit for the ta by pressing save button so why should unsaved details be checked).

**Comment by MadhavaDasari** [comment]
> > Course is getting created ( i.e snackbar will successful msg ) even if the TA's details are not entered properly ..do check it . ( when "save course" button is pressed without pressing "save Ta" )
> > ```
> > * Fields of Ta's are getting outlined ( sm size ) only  when  the edit button is click ( it that intended ? roll_eyes )
> > 
> > * Other things looks good to me +1
> > ```
> > 
> > 
> > >
> 
> * Yes, outlined fields are intentionally shown in small screens only cuz big big boxes in large screens appear vary awkward.
> * I meant even in the small screens  Ta's feilds are getting  outlined only when Edit button is clicked

**Comment by MadhavaDasari** [comment]
> * Yes, outlined fields are intentionally shown in small screens only cuz big big boxes in large screens appear vary awkward.

- I meant even in small screen Ta's feilds are getting outlined only when edit button is clicked

**Pull request merged**